packages/flutter/lib/src/material/drawer.dart

```diff
@@ -261,13 +316,22 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
     _controller.fling(velocity: -1.0);
   }
 
+  bool isOpen() {
```

Comment:
This function doesn't appear to be needed.

Reply:
Removed